src/app/lib/business/environment/environment.dart

```diff
@@ -2,12 +2,14 @@
 enum Environment {
   development,
   staging,
+  generatedStaging,
   production;
 
   @override
   String toString() => switch (this) {
         development => 'Development',
         staging => 'Staging',
+        generatedStaging => 'Generated_Staging',
```

Comment:
Wait why is this required?

Reply:
It's due to this https://github.com/nventive/FlutterApplicationTemplate/blob/da4fd805b84230907291fca2335523c1d662b967/build/azure-pipelines.yml#L41

Reply:
Removed the change in favor of doing the fix on the pipeline instead